docs/new-architecture-app-intro.md

```diff
@@ -27,7 +27,59 @@ Starting from React Native `0.69.0`, you may also need to update the version of
 yarn add react@18.0.0
 ```
 
-### Android specifics
+## Use Hermes
```

Comment:
I moved the `Use Hermes` up because the previous structure was weird.
Previously we had:
- setup Android
- setup Hermes for both ios and android
- setup Android (again)
- setup iOS

The new structure is
- Setup Hermes (for both the platforms)
- Setup Android
- Setup iOS